prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,18 @@
+//defining a variable languages
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+
+//logging the variable
+console.log(languages);
+
+//declaring new variable firstElement
+const firstElement = languages[0];
+console.log('The first element of the array is' + ' ' + firstElement);
+
+const thirdElement = languages[2];
+console.log('The third element of the array is' + ' ' + thirdElement);
+
+const length = languages.length;
+console.log('The length of the array is' + ' ' + length);
+
+const lastElement = languages[5];
```

Comment:
Close! We want to use the length property
`languages[length - 1]`

Reply:
![image](https://github.com/PhilChoe/javascript-prep-0624-code-solutions/assets/172565484/57817177-949c-4069-bb78-91cb41b4b004)

Reply:
![image](https://github.com/PhilChoe/javascript-prep-0624-code-solutions/assets/172565484/46369f9d-ba0b-4708-a162-9ad9ffa91c58)